pkgs/by-name/so/softflowd/package.nix

```diff
@@ -0,0 +1,36 @@
+{
+  stdenv,
+  fetchFromGitHub,
+  autoreconfHook,
+  libpcap,
+  lib,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "softflowd";
+  version = "1.1.0";
+
+  src = fetchFromGitHub {
+    owner = "irino";
+    repo = "softflowd";
+    tag = "softflowd-v${finalAttrs.version}";
+    hash = "sha256-QEudRjOQuv1Y5DbRSxcVkhZqi287KakaZYlKy6I+wtY=";
+  };
+
+  nativeBuildInputs = [
+    autoreconfHook
+  ];
+
+  buildInputs = [
+    libpcap
+  ];
+
+  meta = with lib; {
```

Comment:
please remove with lib; as it increases nix eval times and is now discouraged

Reply:
Good to know